docs/src/whatsnew/latest.rst

```diff
@@ -68,6 +68,9 @@ This document explains the changes made to Iris for this release
    dimension, when the scalar coord provided is already a dim coord.
    (:issue:`4415`, :pull:`5194`)
 
+#. `@HGWright`_ and `@trexfeathers`_ (reviewer) changed the way
+    :class:`CellMethod` are printed to be more CF compliant. (:pull:`5224`)
```

Comment:
```suggestion
   :class:`CellMethod` are printed to be more CF compliant. (:pull:`5224`)
```

The current indent has made this render oddly, like there's a title or something.

![image](https://user-images.githubusercontent.com/40734014/229772767-e5249a92-59ad-4c8c-a01b-905f7dbfe0dd.png)

Reply:
```suggestion
    :class:`~iris.coords.CellMethod` are printed to be more CF compliant. (:pull:`5224`)
```

https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-syntax